config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
Yeah I definitely expect this score and flay to go down a fair amount before merging. I was more so using the number of node types as a proxy for increased complexity in general.